src/mqtt/mqtt_test.h

```diff
@@ -50,6 +50,13 @@ typedef struct MqttTestParam
  */
 void SetupMqttTestParam( MqttTestParam_t * pTestParam );
 
+/**
+ * @brief Customers need to implement this function for MQTT test to set 
+ * MQTTGetCurrentTimeFunc_t in MQTT_Init. This should be an Application 
+ * provided function to query the current time in milliseconds.
+ */
+uint32_t MqttTestGetTimeMs( void );
```

Comment:
This function should be passed by SetupMqttTestParam as this is a parameter in MQTT_Init, [MQTTGetCurrentTimeFunc_t](https://www.freertos.org/Documentation/api-ref/coreMQTT/docs/doxygen/output/html/group__mqtt__callback__types.html#gae3bea55b0e49e5208b8c5709a5ea23aa) getTimeFunction. The test tries to verify the same getTimeFunction used in MQTT demo.